project/plugins.sbt

```diff
@@ -1,9 +1,10 @@
 logLevel := Level.Warn
 
-resolvers += "Typesafe repository" at "https://dl.bintray.com/typesafe/maven-releases/"
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.6")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.9")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
 
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.9.7")
+libraryDependencies += "org.vafer" % "jdeb" % "1.6" artifacts (Artifact("jdeb", "jar", "jar"))
```

Comment:
Just out of interest – why is this now necessary?

Reply:
Play includes a plugin `sbt-native-packager` which is used to package up the app for deployment.

Play 2.5.9 uses `sbt-native-packager` v1.0.3 which depends on `org.vafer.jdeb` to produce Debian packages.  

Play 2.6.0 upgraded to `sbt-native-packager` v1.2.0, and this version has `org.vafer.jdeb` marked as `provided`.